drivers/fuel_gauge/sbs_gauge/emul_sbs_gauge.c

```diff
@@ -100,6 +103,37 @@ static int emul_sbs_gauge_reg_read(const struct emul *target, int reg, int *val)
 	return 0;
 }
 
+static int emul_sbs_gauge_block_read(const struct emul *target, int reg, char *val)
+{
+	char mfg[] = "ACME";
+	char dev[] = "B123456";
+	char chem[] = "LiPO";
+	struct sbs_gauge_manufacturer_name *mfg_name = (struct sbs_gauge_manufacturer_name *)val;
+	struct sbs_gauge_device_name *dev_name = (struct sbs_gauge_device_name *)val;
+	struct sbs_gauge_device_chemistry *dev_chem = (struct sbs_gauge_device_chemistry *)val;
+
+	switch (reg) {
+	case SBS_GAUGE_CMD_MANUFACTURER_NAME:
+		mfg_name->manufacturer_name_length = 4;
```

Comment:
Let's use `sizeof(mfg) - 1` and so forth in case someone changes it later and subsequently forgets to change these numbers.

I know I've done something like that before.